docker-compose-develop.yml

```diff
@@ -3,7 +3,7 @@ services:
   develop:
     build: .
     ports:
-      - "30505:30505"
+      - "9000:9000"
```

Comment:
Localstack looks for the MS on [30505](https://github.com/resource-watch/api-infrastructure/blob/production/terraform-k8s-infrastructure/modules/k8s_microservice_routing/microservices/authorization/main.tf#L14) for dependent microservices to access it although 30505:30505 probably never worked as the Dockerfile only exposes 9000.  Setting this to 30505:9000 would work with localstack and mimick the cloud setup.